canonical/store-components
# added star ratings to card component

## Done
added star ratings to card component

## QA
- Go to https://store-components-26.demos.haus/?path=/story/packagecard--ratings
- See if there are star ratings added to a card 

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-2233
